modules/apps/account/account-service/src/main/java/com/liferay/account/internal/upgrade/v1_0_2/UpgradeRoleName.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.account.internal.upgrade.v1_0_2;
+
+import com.liferay.account.constants.AccountRoleConstants;
+import com.liferay.portal.kernel.service.RoleLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * @author Pei-Jung Lan
+ */
+public class UpgradeRoleName extends UpgradeProcess {
+
+	public UpgradeRoleName(RoleLocalService roleLocalService) {
+		_roleLocalService = roleLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		runSQL(
+			"delete from Role_ where name = '" +
+				AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_ADMINISTRATOR +
+					"'");
+
+		_updateRoleName(
+			AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_ADMINISTRATOR,
+			AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_POWER_USER);
+
+		_updateRoleName(
+			AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_MANAGER,
+			AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_OWNER);
+
+		_updateRoleName(
+			AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_MEMBER,
+			AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_USER);
+	}
+
+	private void _updateRoleName(String oldName, String newName)
+		throws SQLException {
+
+		try (PreparedStatement ps = connection.prepareStatement(
+				"update Role_ set name = ?, title = NULL where name = ?")) {
+
+			ps.setString(1, oldName);
+			ps.setString(2, newName);
```

Comment:
I think the parameter names are backwards here. Either that or the calls switch them. My guess is that this code performs correctly, but reading it is a little confusing. 

We need to swap the arguments for each call (lines 40-50), then change these lines to:
```java
ps.setString(1, newName);
ps.setString(2, oldName);
```

Reply:
oops, you're right. I messed up with the parameter order.